hydra-node/hydra-node.cabal

```diff
@@ -1,16 +1,15 @@
-cabal-version:      3.0
-name:               hydra-node
-version:            0.9.0
-synopsis:           The Hydra node
-author:             IOG
-copyright:          2022 IOG
-license:            Apache-2.0
+cabal-version: 3.0
+name:          hydra-node
+version:       0.9.0
+synopsis:      The Hydra node
+author:        IOG
+copyright:     2022 IOG
+license:       Apache-2.0
 license-files:
   LICENSE
   NOTICE
 
-extra-source-files: README.md
```

Comment:
Perhaps we should keep the `README.md` in order to ensure the package is valid?

Reply:
It also works if I remove this line and I figured this will be merged quicker if I do that compared to writing a proper readme for hydra-node.